### PR TITLE
Etn/misc refactor

### PR DESCRIPTION
**Pull request by ed-nykaza**
@pazaan no rush, but here are some very minor edits for your review

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -11,21 +11,22 @@
...
+    parse_report_and_run, parse_dictionary_from_previous_run
+)
 
 # %% find the path to the file in the repo
 # uncomment the name of the file you'd like to run
 name = "example_issue_report_1.json"
-# name = "example_issue_report_2.json"
-# name = "example_issue_report_3.json"
-# name = "example_from_previous_run.json"
+#name = "example_issue_report_2.json"
```
Comment (pazaan) [inline example.py]:
```suggestion
# name = "example_issue_report_2.json"
```
Not PEP-8 compliant

File: example.py [hunk 2]
```diff
@@ -11,21 +11,22 @@
...
+#name = "example_issue_report_3.json"
```
Comment (pazaan) [inline example.py]:
```suggestion
# name = "example_issue_report_3.json"
```

File: example.py [hunk 3]
```diff
@@ -11,21 +11,22 @@
...
+#name = "example_from_previous_run.json"
```
Comment (pazaan) [inline example.py]:
```suggestion
# name = "example_from_previous_run.json"
```

**Comment by ed-nykaza** [comment]
done @pazaan

**Pull request merged**